local_costmap:
 # TF frame
 global_frame: odom
 robot_base_frame: base_footprint 
 width: 10 
 height: 10
 resolution: 0.05
 rolling_window: true
  
 plugins:
  - {name: obstacles_layer, type: "costmap_2d::ObstacleLayer"}
  - {name: inflater_layer, type: "costmap_2d::InflationLayer"}
  
 # Layers
 obstacles_layer:
  observation_sources: scan
  scan: {sensor_frame: base_scan, data_type: LaserScan, topic: scan, marking: true, clearing: true}
  track_unknown_space: false
  footprint_clearing_enabled: true
  combination_method: 1

 inflater_layer:
  inflation_radius: 4
  cost_scaling_factor: 35



